filters/auth/oidc.go

```diff
@@ -843,7 +843,8 @@ func (f *tokenOidcFilter) Request(ctx filters.FilterContext) {
 	}
 
 	// saving token info for chained filter
-	ctx.StateBag()[oidcClaimsCacheKey] = container
+	// ctx.StateBag()[oidcClaimsCacheKey] = container
+	SetOIDCClaims(ctx, container.Claims)
```

Comment:
We can not do this because container is used to unmarshal json above which also contains other than claims fields.

Reply:
Sorry this is just a leftover from me trying to get existing tests to validate that SetOIDCClaims works correctly. I'll revert that.

Reply:
> Sorry this is just a leftover from me trying to get existing tests to validate that SetOIDCClaims works correctly. I'll revert that.

Maybe it's better to add a standalone testcase?

Reply:
I modified the oauthGrant filter to use SetOIDCClaims and added a use of oidcClaimsQuery in grant_test.go.